css/settings/_sizings.scss

```diff
@@ -0,0 +1,2 @@
+$row-height: 32px;
```

Comment:
When it comes to variable very specific to a component, e.g. `$table-margin`, it's better to keep the variable  in the same file it's used. When it's used in many places, it makes sense to have the variable in a  "global" file.